src/quickgui/qgsquickcoordinatetransformer.cpp

```diff
@@ -76,7 +76,14 @@ void QgsQuickCoordinateTransformer::setSourceCrs( const QgsCoordinateReferenceSy
 
 void QgsQuickCoordinateTransformer::setTransformContext( const QgsCoordinateTransformContext &context )
 {
-  mCoordinateTransform.setContext( context );
+  mTransformContext = context;
```

Comment:
Should this possibly receive a getter in `QgsCoordinateTransform` and the new getter from this PR forwarded there?

Reply:
Fixed